### DISCUSSION TITLE
Beach Island 🏖 | Sand Castle Competition

### DISCUSSION THREAD

**Discussion by BirbSunflower** [sunflower-land/sunflower-land#2136] [Art]
### Summary 🏖

Do you plan to have a beach area at your farm in the future ?
The winner design of this competition will be used as a decoration at the Beach Island ( aaand 👀 )
Through a quest, players will be able to get this item to place at their farms!

### Request :art: 

- A sand castle or sand sculpture

### Dimensions 📏

- 32x32 pixels

### Requirements 📌

- Your farm ID
- Your discord ID
- A minified version of your art piece
- An example of the art used in the scene below. 
( You should place it on the grid with the correct padding/positioning. Pixel dimensions are 1:1 with the background. )
- Make sure your design follows the [Art Guidelines ](https://docs.sunflower-land.com/contributing/for-designers/art-guidelines)

![image](https://user-images.githubusercontent.com/123522693/215360426-6741924e-b675-476a-9289-5ed893cae2e4.png)

### Rewards 🏆

All winners will receive the same prize - sand castle and a care package.

### Deadline 📅

28th of February

### Disclaimer :rotating_light: 

By contributing to the project you agree to the terms and conditions - https://docs.sunflower-land.com/support/terms-of-service. Your art can be used or modified freely by the project and all community members. 

## Ideas:

![image](https://user-images.githubusercontent.com/123522693/215870768-68c6cfb9-a357-4cba-8ee9-1718f942321c.png)

![image](https://user-images.githubusercontent.com/123522693/215870801-16f693aa-eff9-4d5c-9aef-66cb0fc999fe.png)

### More ideas here 
[Sandcastle competition](https://www.google.com/search?q=sandcastle+competition&tbm=isch&ved=2ahUKEwikqrKWj-78AhWXM7kGHYCtBOYQ2-cCegQIABAA&oq=sandcastle+competition&gs_lcp=CgNpbWcQAzIHCAAQgAQQEzIHCAAQgAQQEzIHCAAQgAQQEzIHCAAQgAQQEzIHCAAQgAQQEzIHCAAQgAQQEzIHCAAQgAQQEzIHCAAQgAQQEzIHCAAQgAQQEzIICAAQBxAeEBM6BAgjECc6CAgAEAgQHhATUOgnWOgnYOcpaAFwAHgAgAHJAYgB_wOSAQUwLjIuMZgBAKABAaoBC2d3cy13aXotaW1nwAEB&sclient=img&ei=JhrXY6T_OJfn5OUPgNuSsA4&bih=726&biw=1536)

**Comment by BirbSunflower** [comment]
## Submission example:
 
 Minified:
   
  ![chest](https://user-images.githubusercontent.com/123522693/215360944-ec28b4bc-0751-401f-88ec-6cd237bc8b68.png)
  
 Preview
  ![example2](https://user-images.githubusercontent.com/123522693/215361102-e10a5c27-a51f-41b9-bcec-da180f363c19.png)
    
  Farm ID: 56
  Discord: Gabi#9387

**Comment by Kiulg** [comment]
### Castle_v1 ![Castle](https://user-images.githubusercontent.com/18270876/216128694-ca16418b-985c-4e76-b437-0bf105551ac6.gif)
![Castle8](https://user-images.githubusercontent.com/18270876/216128757-6f5635fb-441e-4cbc-9d92-c69990f8d9e2.gif)

### Castle_v2 ![Castle2](https://user-images.githubusercontent.com/18270876/216513245-78a28d9d-b1c0-4dfe-8de3-b4542f4b59cb.gif)
![Castle28](https://user-images.githubusercontent.com/18270876/216513258-5ada08c4-fa11-4004-aae7-3822b1eaea59.gif)

### King Crab Castle ![crab](https://user-images.githubusercontent.com/18270876/216885356-3d523857-26b7-4f12-8246-0c6116ec2252.gif)
![crab4](https://user-images.githubusercontent.com/18270876/216885446-bbfeddf9-d855-46c0-a715-34c61d062e8d.gif)



![215360426-6741924e-b675-476a-9289-5ed893cae2e4](https://user-images.githubusercontent.com/18270876/216514855-d8665268-168d-401b-bc10-fd13c9d4ae56.png)

`Farm: 100430`
`Discord: Kiulg#8058`

**Comment by peperol** [comment]
Farm Id 145849
Discord: Peperol#2715

SandCastle 

![castleultim](https://user-images.githubusercontent.com/114354077/217584656-cef3675c-bba3-4929-8c7e-5d803d5f8350.png)



![landcastle (2)](https://user-images.githubusercontent.com/114354077/216721015-90fd8a3b-cae6-4cf4-b8d0-45af4ef775a3.png)

sandcastle with bucket
![Secchiellook-1](https://user-images.githubusercontent.com/114354077/216721257-ec70c35e-9c37-4dfe-84c8-24db491530cb.png)

Sand Siren sculpture
![siok](https://user-images.githubusercontent.com/114354077/216461769-2ee01520-5c89-494c-a12a-e185a5aa1f7d.png)

![Siren ok-1](https://user-images.githubusercontent.com/114354077/216461804-6f5aa9c7-7c2e-4309-8d95-267ae806ed65.png)
 

Neutron Star Collision 
![screen posi](https://user-images.githubusercontent.com/114354077/217323623-0c89f69b-eec9-4be2-b7e5-92862720d8e4.png)

with sea rake
![Siren posi con1](https://user-images.githubusercontent.com/114354077/217324103-d6a30c21-1532-4fe2-ab0c-50c57cbbd8e3.png)

without rake
![Siren posisenza1](https://user-images.githubusercontent.com/114354077/217324294-1663b53f-fc23-4261-be9c-12543111d351.png)

**Comment by FarmerKuenster** [comment]
Plain sand castle 
![sand castle plain](https://user-images.githubusercontent.com/104322749/217061672-d0063a3b-0138-4dfe-b95a-bec213004cb3.png)
![sand castle plain-large](https://user-images.githubusercontent.com/104322749/217061701-33088c09-1e4f-400e-9110-7bbc7b1752b2.png)


Sand castle with decorations
![sand castle with decor](https://user-images.githubusercontent.com/104322749/217061682-f6f865c9-9148-4a79-a427-4108d13b1553.png)
![sand castle with decor-large](https://user-images.githubusercontent.com/104322749/217061716-f8e261f6-7899-4104-b8c8-30349931c44a.png)

![sandcastles on land-export](https://user-images.githubusercontent.com/104322749/217061745-b06ce1df-c059-47d1-8d3b-4459ac6f80e1.png)

Fish sand sculpture
![fish sandcastle](https://user-images.githubusercontent.com/104322749/217058799-bec8f0c0-e537-4e2e-a433-10b37ec062a5.png)

![fish sandcastle-large](https://user-images.githubusercontent.com/104322749/217058825-777bd56f-70c9-43b1-8dbf-ea121b917203.png)

![fish sand castle on land](https://user-images.githubusercontent.com/104322749/217059184-af1d8992-f178-4237-83b7-8d7bb948ce7c.png)


discord: farmerkunester#1626
land ID: #32391

**Comment by BirbSunflower** [comment]
@Kiulg - https://github.com/sunflower-land/sunflower-land/discussions/2136#discussioncomment-4843667
@FarmerKuenster - https://github.com/sunflower-land/sunflower-land/discussions/2136#discussioncomment-4854847

All of these designs have a different perspective than the one we use in the game - top-down view. To learn more about it, please check this guideline's topic :heart: 

https://docs.sunflower-land.com/contributing/for-designers/art-guidelines#top-down-view

**Comment by N1491** [comment]
Minified:
![castle](https://user-images.githubusercontent.com/121005517/218254210-47bc29e3-2a3c-4c3d-961c-a86b8ddf67ea.png)
Preview
![land castle](https://user-images.githubusercontent.com/121005517/218254218-7e9b9493-faf2-452a-95c8-a7177b9a7b0d.png)
Farm ID: 139032 Discord: 𝓙𝓮𝓽𝓱𝓮𝓻 | #¹³⁹⁰³²#1377

**Comment by Rylinn** [comment]
### **Minified version:**
**Goblins..style of a Sandcastle**: **9** frames

![Sandcastle_spritesheet_288x32](https://user-images.githubusercontent.com/6690967/218288307-92074a16-35ef-4453-9347-5e0ec04226ef.png)

![Sandcastle_goblin_640x640](https://user-images.githubusercontent.com/6690967/218288312-b479bb48-75c8-4120-87b1-cf0afee24b6f.gif)

Goblin shovel animation is based on the original artwork by DanielDiggle<3

Farm ID: 154241
Discord: Rylinn#9925

**Comment by minthwe** [comment]
Farm id :128281
Discord.  : Superman#8800

minified version
Castle 
![1B35DB5C-708A-443D-96F4-1637BCD29067](https://user-images.githubusercontent.com/29115241/219684787-87960f70-7214-4c59-ae9f-ad2327471a4c.png)


![547F2792-F6E4-4BE3-96DE-8BDC2965F879](https://user-images.githubusercontent.com/29115241/219684298-4fcb9921-461d-42c8-9757-8b2e98483345.png)

SFL sword sculpture 
![0E0B4FC9-3136-4BA6-BD06-60434FC4428E](https://user-images.githubusercontent.com/29115241/219830379-d0d25d97-d6e0-4434-ab3a-90008c028cd1.png)
![127868CC-13B6-45E0-8EF0-1955BA5571B9](https://user-images.githubusercontent.com/29115241/219830386-3458b1dd-0013-472b-956e-64a78af2f8d4.png)

**Comment by Mehran7583** [comment]
farm ID : 20401
Your discord ID : mehran7583#2642

minified version :
![sand castle (32-32)](https://user-images.githubusercontent.com/108087644/219789533-f50beaf8-2e2f-454d-9c80-5247942e492e.png)


![sand castle](https://user-images.githubusercontent.com/108087644/219788904-5349e5da-f60a-4b74-8834-7cdd7280493b.png)

**Reply by RezA7X** [reply]
perfect

**Reply by sabryder** [reply]
I love it❤️😍😍😍
And i like the idea of having 2 flags (goblin and human) on top of 2 towers👌🏻❤️

**Reply by V1to0** [reply]
I fell in love with this castle, I will have it soon.🥹😍👌

**Reply by Majidgnd** [reply]
Is very good💥💥

**Reply by immilad** [reply]
awesome & perfect...good luck👍

**Reply by Erfansoufi** [reply]
its so coolll

**Comment by JcEii** [comment]
Preview:
![sample](https://user-images.githubusercontent.com/105348036/219820833-2c63698d-a708-40b0-b4a7-7da8ca6676a7.gif)

Holding Sunflower 
![sample1](https://user-images.githubusercontent.com/105348036/219821038-08535ccb-6f73-4853-be1e-0a736566849c.png)

Malacañang Palace
![sample2](https://user-images.githubusercontent.com/105348036/219821040-ad89df4e-4270-44fe-91fb-19777a1ec6c6.png)

Palace of us
![sample3](https://user-images.githubusercontent.com/105348036/219821041-dd3b38df-ec73-49e2-93f9-5ef9db3ebe19.png)

Megalodon's jaw
![sample4](https://user-images.githubusercontent.com/105348036/219821042-10564d38-7cff-4468-a1e1-9573088e3c2c.png)

Best castle in the world
![sample5](https://user-images.githubusercontent.com/105348036/219821043-608cb9fd-3504-420a-b012-6a40f840436c.png)

burjer
![sample6](https://user-images.githubusercontent.com/105348036/219821044-28332af7-efab-48da-8bd5-a3290bbda0eb.png)


Minified:
![Sprite-0001](https://user-images.githubusercontent.com/105348036/219820846-dbfce156-7642-4c2b-ae58-4e3a33e0532e.png)
![Sprite-0002](https://user-images.githubusercontent.com/105348036/219820850-6b7c4640-6013-45be-8f65-74195711050e.png)
![Sprite-0003](https://user-images.githubusercontent.com/105348036/219820853-993d2bba-5a8c-433f-8db9-cce4630841f9.png)
![Sprite-0004](https://user-images.githubusercontent.com/105348036/219820855-bb7f3127-f19f-4729-965d-2dfa6d4b85ca.png)
![Sprite-0005](https://user-images.githubusercontent.com/105348036/219820858-4120e544-3f51-4f37-9bb4-ee5470e18fbe.png)
![Sprite-0006](https://user-images.githubusercontent.com/105348036/219820859-c02e758b-4d41-465a-9638-5fb55f6e372d.png)



Farm ID: 15747
Discord: Jc Eii #3821

**Comment by N1491** [comment]
Minified:
![Human Castle](https://user-images.githubusercontent.com/121005517/220260628-1f4beb0b-4776-4de8-b382-fab85dcc2fdb.png)
Preview:
![219820853-993d2bba-5a8c-433f-8db9-cce4630841f9 (1)](https://user-images.githubusercontent.com/121005517/220260673-f724bc7c-2ebb-4c0c-ab33-a5fdba598dd2.png)
Farm ID: 139032 Discord: 𝓙𝓮𝓽𝓱𝓮𝓻 |#¹³⁹⁰³²#1377

**Comment by shinooon** [comment]
Preview:
![sand castle](https://user-images.githubusercontent.com/109294090/220875405-6768743d-5eac-4c31-baa5-2477e0a1b7bc.gif)
Minified:
![sand castle](https://user-images.githubusercontent.com/109294090/220876292-a4333be3-7f2e-4a1a-8952-4c4fad653f5e.png)
with flag
![sand castle with flag](https://user-images.githubusercontent.com/109294090/220876355-df68fb49-0734-4fe4-b1e6-88ce49bced40.png)


farm id: 51738
discord id: shinon#4931

**Comment by Vergelsxtn** [comment]
**Castle in Mountain** 
Minified:
![sand-castle-vergelsxtn](https://user-images.githubusercontent.com/99605039/221460041-40f1e0f0-bb07-4107-b45b-9369d0b066f2.png)

Preview:
![sand-castle-prev1](https://user-images.githubusercontent.com/99605039/221460046-0348fb2b-f842-4e7d-ac1e-fae839dc8c77.png)

**Castle with wall**
Minified:
![sand-castle-vergelsxtn2](https://user-images.githubusercontent.com/99605039/221524878-08222c4e-dbd6-40f9-a45e-a8cc51973552.png)

Preview:
![prev](https://user-images.githubusercontent.com/99605039/221524931-7a9e1ad7-744b-4a43-9385-d51b7b21b644.png)

Farm ID: 63
Discord: Vergelsxtn#8727
